www/js/controller.js

```diff
@@ -168,10 +168,16 @@ const winMenu = [
         enabled: false,
       },
       ...updateMenu,
+      {
+        label: 'Guide...',
```

Comment:
Could we please put this in a Help menu?

Reply:
I think this is already done!

Reply:
Yes, you're right. Sorry